### PR TITLE
Ignored unnecessary files

### PR DESCRIPTION
**Pull request by SrinivasanTarget**
@imurchie @scottdixon Please review

### CODE REVIEW

**Comment by SrinivasanTarget** [comment]
https://github.com/appium/appium/issues/6325

File: README.md [hunk 1]
```diff
@@ -3,8 +3,8 @@ appium-adb
 
 [![NPM version](http://img.shields.io/npm/v/appium-adb.svg)](https://npmjs.org/package/appium-adb)
 [![Downloads](http://img.shields.io/npm/dm/appium-adb.svg)](https://npmjs.org/package/appium-adb)
-[![Dependency Status](https://david-dm.org/appium/appium-adb/2.0.svg)](https://david-dm.org/appium/appium-adb/2.0)
-[![devDependency Status](https://david-dm.org/appium/appium-adb/2.0/dev-status.svg)](https://david-dm.org/appium/appium-adb/2.0#info=devDependencies)
+[![Dependency Status](https://david-dm.org/appium/appium-adb/2.0.svg)](https://david-dm.org/appium/appium-adb)
+[![devDependency Status](https://david-dm.org/appium/appium-adb/2.0/dev-status.svg)](https://david-dm.org/appium/appium-adb/#info=devDependencies&view=table)
```
Comment (scottdixon) [inline README.md]: Please check the SVG urls, as in: https://david-dm.org/appium/appium-adb.svg

File: .npmignore [hunk 2]
```diff
@@ -16,3 +16,4 @@ npm-debug.log
 node_modules
 coverage
 coverage-e2e
+test/
```
Comment (scottdixon) [inline .npmignore]: missing a new line

**Comment by scottdixon** [comment]
Thanks! A couple of comments.

**Comment by SrinivasanTarget** [comment]
Yup updated now

File: .npmignore [hunk 3]
```diff
@@ -16,3 +20,5 @@ npm-debug.log
 node_modules
 coverage
 coverage-e2e
+
+test/
```
Comment (scottdixon) [inline .npmignore]: New line should be at the end of the file - sorry to nitpick!

Also, can you please squash your `updated dependency path` commits into one?

Comment (SrinivasanTarget) [inline .npmignore]: Sure added few more to ignore now

**Comment by SrinivasanTarget** [comment]
@scottdixon Squashed now

File: .npmignore (hunk 2 above)
Comment (SrinivasanTarget) [inline .npmignore]: Do i need to add a new line before?

**Comment by imurchie** [comment]
There is more we can ignore. The `README.md` file, for one, as well as the origin source and the built tests. None of those are needed by users of the package.

File: .npmignore (hunk 2 above)
Comment (scottdixon) [inline .npmignore]: Not in this case. All files [should _end_ with a new line](https://robots.thoughtbot.com/no-newline-at-end-of-file).

**Comment by SrinivasanTarget** [comment]
@imurchie README.md cannot be ignored, https://docs.npmjs.com/misc/developers

File: .npmignore (hunk 2 above)
Comment (SrinivasanTarget) [inline .npmignore]: Ok

**Comment by imurchie** [comment]
Ah. Cool.

**Comment by SrinivasanTarget** [comment]
What does that origin source mean here? Is it the jar files we can exclude?

**Comment by imurchie** [comment]
No, I mean all the un-transpiled JavaScript files. I'm not sure how things like stack traces will work. It needs some testing.

**Comment by SrinivasanTarget** [comment]
Hmm didn't get completely help me with that

**Comment by SrinivasanTarget** [comment]
@imurchie ping

**Pull request closed without merging**